wal/wal.go

```diff
@@ -289,6 +294,7 @@ func (w *WAL) Repair(origErr error) error {
 	if !ok {
 		return errors.Wrap(origErr, "cannot handle error")
 	}
+	w.walCorruptionsTotal.Inc()
```

Comment:
Is it safe to assume that `Repair` is always called after corruption? (Should not regress in future as this is a separate package)

Reply:
sorry I didn't understand what you mean. 
I think the metric is to notify for corruption, regardless if it has been repaired or not.

Reply:
Yes, so it should not happen in the future that we encounter a corruption but don't call `Repair` due to some reasons. In that case, corruption is not counted. Would it be better to increment the counter right when we encounter corruption?

Reply:
yeah I see you point. I moved it.